sources/packages/web/src/components/common/OfferingSummary.vue

```diff
@@ -156,19 +248,17 @@ export default defineComponent({
       return allowOfferingEdit.value ? "Edit" : "View";
     });
 
-    const goToAddNewOffering = () => {
-      if (isInstitutionUser.value) {
-        router.push({
-          name: InstitutionRoutesConst.ADD_LOCATION_OFFERINGS,
-          params: {
-            locationId: props.locationId,
-            programId: props.programId,
-            clientType: ClientIdType.Institution,
-          },
-        });
-      }
-    };
+    /**
+     * Debounced version of the search function with a 2000ms (2 second) delay
+     * */
+    const debouncedSearch = debounce(() => {
+      searchOfferingTable();
+    }, 2000); // 2000 milliseconds delay
```

Comment:
The current behaviour to trigger a search is on enter or press search. Do we require this granular level of search on server side pagination?

Reply:
Actually the current behaviour is: search is triggered by enter or by clicking search or by clicking in intensity toggle buttons.

The debounce is just a clever way to search by value change, I mostly introduced this because of the date picker object, since you're using the mouse to pick the dates there is no easy way to search other than clicking the button. 

I did not understand the last part of your comment about the server side pagination, Im happy to discuss more about this!